internal/image/builder.go

```diff
@@ -111,50 +111,57 @@ func NewImageBuilder(sourceID uuid.UUID) *ImageBuilder {
 	return imageBuilder
 }
 
-func (b *ImageBuilder) Generate(ctx context.Context, w io.Writer) (int, error) {
+func (b *ImageBuilder) Size() (int, isoeditor.ImageReader, error) {
```

Comment:
`ImageReader` is never returned. where do you plan to use it?

Reply:
Right! It's not needed.